Doc/cmus.txt

```diff
@@ -1141,7 +1145,7 @@ show_playback_position (true)
 show_remaining_time (false)
 	Display remaining time instead of elapsed time.
 
-shuffle (false)
+shuffle (off) [off, tracks, albums]
 	Play in shuffled order. Shuffle works in the library views (1-2) and
 	playlist view (3).
```

Comment:
To make this less ambiguous, it should say something about `tracks` being any track within the entire lib rather than just the current album (speaking of this, an `album-tracks` option might be useful). It should also mention that `albums` only chooses the first track of a random album after the current one has ended. To prevent confusion, it might be a good idea to mention that the `albums` option does nothing if the current `aaa_mode` is `album`.

Reply:
`album-tracks` sounds like `aaa_mode=album` + `shuffle=tracks`, though I doubt many people really know what `aaa_mode` is (until it's tripped them up enough, at least).